src/SDKs/Network/Management.Network/Generated/Models/VirtualNetworkGatewaySkuTier.cs

```diff
@@ -21,6 +21,9 @@ public static class VirtualNetworkGatewaySkuTier
         public const string HighPerformance = "HighPerformance";
         public const string Standard = "Standard";
         public const string UltraPerformance = "UltraPerformance";
+        public const string VpnGw1 = "VpnGw1";
```

Comment:
This is the swagger PR : https://github.com/Azure/azure-rest-api-specs/pull/1130.

I pulled the latest sdk branch - I am not sure of the versioning. I did not change anything except for the three generated code files in the PR.